src/TwigComponent/doc/index.rst

```diff
@@ -840,6 +840,76 @@ The ``with`` data is what's mounted on the component object.
 .. note::
 
     Embedded components *cannot* currently be used with LiveComponents.
+    
+Twig Component syntax
+-----------------
+    
+.. versionadded:: 2.9
+
+    This syntax has been introduced in 2.9, and it's still experimental.
+    
+Twig Component came with a new syntax to ease the readability of your template:
+
+.. code-block:: html+twig
+
+    {# templates/some_page.html.twig #}
+    <twig:table :headers="['key', 'value']" :data="[[1, 2], [3, 4]]">
+        <twig:block name="th_class">
+            {{ parent() }}
+        </twig:block>
+         <twig:block name="td_class">
+            {{ parent() }}
+        </twig:block>
+        <twig:block name="footer">
+            <div class="data-table-footer">
+               My footer
+            </div>
+        </twig:block>
+    </twig:table>
```

Comment:
Let's use a simple `Alert` example - it's a bit simpler, so easier to understand. And remember to call the component `Alert` - we now match the class name.